slider/library.py

```diff
@@ -53,7 +53,7 @@ def keys(self):
 if sys.platform.startswith('win'):
     def sanitize_filename(name):
         for invalid_character in r':*?"\/|<>[]':
-            name.replace(invalid_character, '')
```

Comment:
sorry, I totally agree. Some of the characters in the string are in fact valid for windows based on the original screen shot.